javascript/javascript1/week1/1.js

```diff
@@ -0,0 +1,121 @@
+// Age-IFy excericse
+
+const yearOfBirth = 1987;
+let yearFuture = 2027;
+const age = yearFuture - yearOfBirth;
+
+console.log(
+  "You will be" + " " + age + " " + "years old in" + " " + yearFuture + "."
```

Comment:
Check Template literals and how you can write these statements in a more elegant way https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals